### PR TITLE
docs ja block types fix [ci skip]

### PR DESCRIPTION
**Pull request by softark**

### CODE REVIEW

**Comment by cebe** [comment]
Awesome, thank you!

File: docs/guide-ja/concept-behaviors.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ class MyBehavior extends Behavior
 
 このクラスはビヘイビアなので、コンポーネントにアタッチされると、そのコンポーネントは `prop1` と `prop2` プロパティ、それと `foo()` メソッドを持つようになります。
 
-> Tip: ビヘイビア内から、[[yii\base\Behavior::owner]] プロパティを介して、ビヘイビアをアタッチしたコンポーネントにアクセスすることができます。
+> Tip:: ビヘイビア内から、[[yii\base\Behavior::owner]] プロパティを介して、ビヘイビアをアタッチしたコンポーネントにアクセスすることができます。
```
Comment (rugabarbo) [inline docs/guide-ja/concept-behaviors.md]: `Tip::` -> `Tip:`

File: docs/guide-ja/test-overview.md [hunk 2]
```diff
@@ -42,7 +42,7 @@
 
 通常は、長い期間で見れば、かなり時間を節約する効果があります。
 
-> Tip: ソフトウェアの要求仕様の取り纏めと対象事物のモデリングに関する原則について更に知りたい場合は、[ドメイン駆動設計 (DDD)](http://ja.wikipedia.org/wiki/%E3%83%89%E3%83%A1%E3%82%A4%E3%83%B3%E9%A7%86%E5%8B%95%E8%A8%AD%E8%A8%88) を学習するのが良いでしょう。
+> Tip:: ソフトウェアの要求仕様の取り纏めと対象事物のモデリングに関する原則について更に知りたい場合は、[ドメイン駆動設計 (DDD)](http://ja.wikipedia.org/wiki/%E3%83%89%E3%83%A1%E3%82%A4%E3%83%B3%E9%A7%86%E5%8B%95%E8%A8%AD%E8%A8%88) を学習するのが良いでしょう。
```
Comment (rugabarbo) [inline docs/guide-ja/test-overview.md]: `Tip::` -> `Tip:`

File: docs/guide-ja/test-overview.md [hunk 3]
```diff
@@ -29,7 +29,7 @@
 完了すれば、別の機能または改良のために、このプロセスを再び繰り返します。
 既存の機能が変更される場合は、テストも変更されなければなりません。
 
-> Tip: 多数の小さくて単純なイテレーションを繰り返すために時間を取られていると感じる場合は、テストシナリオのカバー範囲を広くして、テストを再度実行するまでの作業量を増やしてみてください。
+> Tip:: 多数の小さくて単純なイテレーションを繰り返すために時間を取られていると感じる場合は、テストシナリオのカバー範囲を広くして、テストを再度実行するまでの作業量を増やしてみてください。
```
Comment (rugabarbo) [inline docs/guide-ja/test-overview.md]: `Tip::` -> `Tip:`

**Comment by cebe** [comment]
@rugabarbo these are already fixed in the merged version: https://github.com/yiisoft/yii2/commit/059bc72e48994b34f7c624cdacbeab6f16877c1b

**Pull request closed without merging**